src/api/utils/InterceptorManager.js

```diff
@@ -0,0 +1,38 @@
+class InterceptorManager {
+  handlers = [];
+
+  use(fullfill, failure) {
```

Comment:
I'm not sure if i'm understanding this correctly but do we need to call `use` this way? I don't think you are using `failure` right now.

Is it an error or is it because you didn't believe it was worth it adding a failure interceptor to the base?

Reply:
I just didn't use it on the base